src/components/SuggestionTags/hooks/useSuggestionTags.tsx

```diff
@@ -0,0 +1,210 @@
+import { DCA_HIGH_PRICE_IMPACT, JLP_MINT, USDC_MINT, USDT_MINT } from 'src/constants';
+import { TokenInfo } from '@solana/spl-token-registry';
+import { QuoteResponse } from '@jup-ag/react-hook';
+import { useHeliusDASQuery } from './useHeliusDasQuery';
+import { useLstApyFetcher } from './useLstApy';
+import { useBirdeyeRouteInfo } from './useSwapInfo';
+import { useMemo } from 'react';
+import { SystemProgram } from '@solana/web3.js';
+import Decimal from 'decimal.js';
+import JSBI from 'jsbi';
+import { checkIsUnknownToken } from 'src/misc/tokenTags';
+import { UnknownTokenSuggestion } from '../Tags/UnknownTokenSuggestion';
+import { LSTSuggestion } from '../Tags/LSTSuggestion';
+import { AuthorityAndDelegatesSuggestion } from '../Tags/AuthorityAndDelegatesSuggestion';
+import { TransferTaxSuggestion } from '../Tags/TransferTaxSuggestion';
+import { PriceWarningSuggestion } from '../Tags/PriceWarningSuggestion';
+import { DCASuggestion } from '../Tags/DCASuggestion';
+import { extractTokenExtensionsInfo } from '../Tags/Token2022Info';
+import { useUSDValue } from 'src/contexts/USDValueProvider';
```

Comment:
Hi, we only need to support a subset of warnings as mentioned in the ticket.
```
AuthorityAndDelegatesSuggestion
PriceWarningSuggestion
TransferTaxSuggestion
UnknownTokenSuggestion
```

Reply:
remember to drop unused files

Reply:
removed used files